Code/GraphMol/Substruct/vf2.hpp

```diff
@@ -106,8 +126,9 @@ node_id *SortNodesByFrequency(const Graph *g) {
   for (unsigned int i = 0; i < vect.size(); i += run) {
     for (run = 1; i + run < vect.size() && vect[i + run].in == vect[i].in &&
                   vect[i + run].out == vect[i].out;
-         ++run)
+         ++run) {
```

Comment:
This one is kind of silly looking, but the original code is also not so nice looking